types/react/index.d.ts

```diff
@@ -1856,6 +1856,10 @@ declare namespace React {
         draggable?: Booleanish | undefined;
         hidden?: boolean | undefined;
         id?: string | undefined;
+        /**
+         * @see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert
```

Comment:
Do we need a comment in here? None of the other attributes have comments.

Reply:
> Do we need a comment in here?

It's JSDOC which means it appears in editors supporting it. This helps people understand what this attribute is for.


> None of the other attributes have comments.

I don't know why for each. For some there's no useful MDN article. For some people might not have been aware how useful JSDOC can be